tools/pkglint/lib/rules.ts

```diff
@@ -617,6 +617,38 @@ export class NoPeerDependenciesMonocdk extends ValidationRule {
   }
 }
 
+/**
+ * Validates that the same version of `constructs` is used wherever a dependency
+ * is specified, so that they must all be udpated at the same time (through an
+ * update to this rule).
+ *
+ * Note: v1 and v2 use different versions respectively.
+ */
+export class ConstructsConsistentDependencyVersion extends ValidationRule {
+  public readonly name = 'peer-deps/constructs';
```

Comment:
since you're checking all dependencies

```suggestion
  public readonly name = 'deps/constructs';
```